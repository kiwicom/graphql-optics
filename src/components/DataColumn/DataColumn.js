import React from 'react'

const style = {
  height: '100vh',
  overflowY: 'auto',
  paddingRight: '40px',
  width: '30%',
}

export default ({ children }) => <div style={style}>{children}</div>
