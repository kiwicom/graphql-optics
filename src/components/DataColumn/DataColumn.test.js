import React from 'react'
import DataColumn from './DataColumn'
import renderer from 'react-test-renderer'

it('renders children', () => {
  const children = <div>Hello!</div>
  const tree = renderer.create(<DataColumn>{children}</DataColumn>).toJSON()
  expect(tree).toMatchSnapshot()
})
