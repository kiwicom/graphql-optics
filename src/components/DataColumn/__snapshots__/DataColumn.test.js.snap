// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders children 1`] = `
<div
  style={
    Object {
      "height": "100vh",
      "overflowY": "auto",
      "paddingRight": "40px",
      "width": "30%",
    }
  }
>
  <div>
    Hello!
  </div>
</div>
`;
